package.json

```diff
@@ -17,10 +17,10 @@
     "@ember/optional-features": "~0.7.0",
     "babel-eslint": "^10.0.1",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.7.1",
+    "ember-cli": "~3.8.1",
     "ember-cli-app-version": "^3.2.0",
     "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-eslint": "^5.0.0",
```

Comment:
Is this downgrade needed?

Reply:
No, but because of the ^ it just matches the latest 5.x.x, so this makes no difference